starter_code/express-spotify-app/app.js

```diff
@@ -0,0 +1,123 @@
+//https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes
+//https://www.npmjs.com/package/spotify-web-api-node
+
+
+var SpotifyWebApi = require('spotify-web-api-node');
+
+var clientId = '233be0ff6eec4286a17021b15024c094',
```

Comment:
**This are sensitive data**, so I recommend always save as a enviorment variable. We already learn this, so in your future project always add this kind of data into your `.env` file, and make sure that your `.env` file is mentioned in the `.gitignore` file before to make any commit :grin: 

Then I will told you a funny not so funny story about this :wink: